src/paradigma/tremor/tremor_analysis.py

```diff
@@ -127,29 +125,14 @@ def aggregate_tremor_power(tremor_power: pd.Series, config: TremorQuantification
     return tremor_power_median, tremor_power_90th_perc, tremor_power_mode
 
 
-def quantify_tremor(df: pd.DataFrame, config: TremorQuantificationConfig, utc_start_time: str):
+def quantify_tremor(df: pd.DataFrame, config: TremorQuantificationConfig):
 
-    # Create time array in local time zone
-    utc_start_time = datetime.strptime(utc_start_time, "%Y-%m-%dT%H:%M:%SZ").replace(tzinfo=pytz.UTC) # convert to datetime
-    local_start_time = utc_start_time.astimezone(pytz.timezone("Europe/Amsterdam"))
-    time = [local_start_time + timedelta(seconds=x) for x in df.time]
-
-    # determine valid days
-    daytime_hours = range(config.daytime_hours_lower_bound,config.daytime_hours_upper_bound)
-    nr_windows_per_day = Counter([dt.date() for dt in time if dt.hour in daytime_hours])
-    valid_days = [day for day, count in nr_windows_per_day.items() 
-                  if count * config.window_length_s / 3600 >= config.valid_day_threshold_hr]
-    nr_valid_days = len(valid_days)
-
-    # remove windows during non-valid days, non-daytime hours and non-tremor arm movement
-    df_filtered = df.loc[[dt.date() in valid_days for dt in time]]
-    df_filtered = df_filtered.loc[[dt.hour in daytime_hours for dt in time]]
-    nr_windows_daytime = df_filtered.shape[0] # number of windows during daytime on valid days
-    df_filtered = df_filtered.loc[df_filtered.low_freq_power <= config.movement_threshold]
-    nr_windows_daytime_rest = df_filtered.shape[0] # number of windows during daytime on valid days without non-tremor arm movement
+    # remove windows with detected non-tremor movements
```

Comment:
Perhaps you could specify that *above* the movement threshold, you expect there to be movement that interferes(?) with the tremor quantification. I'm not sure about the specifics, but it might make sense to clarify this subsetting.